packages/protocol/contracts/thirdparty/onchainRA/AutomataDcapV3Attestation.sol

```diff
@@ -134,7 +134,8 @@ contract AutomataDcapV3Attestation is IAttestation {
     {
         return status == TCBInfoStruct.TCBStatus.OK
```

Comment:
@smtmfft please make improvements here: https://github.com/taikoxyz/automata-dcap-v3-attestation

Reply:
In the future, please release https://github.com/taikoxyz/automata-dcap-v3-attestation then in taiko-mono, we can use a "tag = v.1.0.0" to specify what code to use.